src/compute-model-statistics/src/test/scala/VerifyComputeModelStatistics.scala

```diff
@@ -81,10 +95,10 @@ class VerifyComputeModelStatistics extends TestBase {
 
     val evaluatedData = new ComputeModelStatistics().transform(datasetWithScores)
     val firstRow = evaluatedData.first()
-    assert(firstRow.get(0) == 0.0)
-    assert(firstRow.get(1) == 0.0)
-    assert(firstRow.get(2) == 1.0)
-    assert(firstRow.get(3) == 0.0)
+    assert(tolEq.areEquivalent(firstRow.get(0).asInstanceOf[Double], 0.0))
```

Comment:
can replace with "===" if you make tolEq implicit

Reply:
done